src/components/Stories.js

```diff
@@ -21,10 +21,11 @@ const Stories = props => {
   }, [props.storytype])
 
   const loadItems = (pageNumber) => {
+
     const pageItems = paginate(stories, pageNumber, 10) //10 items per page
     setrenderlist([...renderlist, ...pageItems.data])
     setinitialLoad(false)
-    sethasMoreItems(pageItems.currentPage !== pageItems.totalPages)
+    sethasMoreItems((pageItems.currentPage !== pageItems.totalPages) && pageItems.data.length !== 0)
```

Comment:
Looks fine and it is working. But as per paginate docs only that check should be enough and it was working fine before I am not sure what went wrong. 
https://www.npmjs.com/package/paginate-array

Reply:
Does the API return correct data set ? I'll check it and add a new issue for that if needed. For now let's keep it as this for the further development of the app.